.github/workflows/release.yml

```diff
@@ -10,15 +10,86 @@ jobs:
     env:
       GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
     steps:
-    - uses: actions/checkout@v3
-      with:
-        fetch-depth: 0 # Disable shallow checkout
-    - uses: ./.github/actions/setup-go
-    - run: go run . --help > cli-reference.txt
-    - run: go run testutil/genchangelog/main.go
-    - uses: softprops/action-gh-release@v1
-      with:
-        draft: true
-        files: cli-reference.txt
-        body_path: changelog.md
-        token: ${{ secrets.RELEASE_SECRET }}
+      - name: Checkout repository
+        uses: actions /checkout@v4
+        with:
+          fetch-depth: 0 # Disable shallow checkout
+
+      - name: Setup Go environment
+        uses: ./.github/actions/setup-go
+
+      - name: Generate CLI reference
+        run: go run . --help > cli-reference.txt
+
+      - name: Generate changelog
+        run: go run testutil/genchangelog/main.go
+
+      - name: Create GitHub release draft
+        uses: softprops/action-gh-release@v1
+        with:
+          draft: true
+          files: cli-reference.txt
+          body_path: changelog.md
+          token: ${{ secrets.RELEASE_SECRET }}
+
+  extract-tag:
+    runs-on: ubuntu-latest
+    outputs:
+      tag_name: ${{ steps.get_tag_name.outputs.tag_name }}
+    steps:
+      - name: Extract tag name
+        id: get_tag_name
+        run: echo "::set-output name=tag_name::${GITHUB_REF##*/}"
+
+  create-pr-obol-infrastructure:
```

Comment:
Let's discuss the infra repo updates on slack, because we do not deploy the perf cluster from there any more, also we deploy the p2p dv relays from there.